docs/operating-scylla/procedures/cluster-management/add-node-to-cluster.rst

```diff
@@ -5,49 +5,73 @@ Adding a New Node Into an Existing ScyllaDB Cluster (Out Scale)
 When you add a new node, other nodes in the cluster stream data to the new node. This operation is called bootstrapping and may
 be time-consuming, depending on the data size and network bandwidth. If using a :ref:`multi-availability-zone <faq-best-scenario-node-multi-availability-zone>`, make sure they are balanced.
 
-.. note::
 
-    Adding a new node requires at least a quorum of nodes in a cluster to be available. 
-    If the quorum is lost, it must be restored before a new node is added. 
-    See :doc:`Handling Node Failures </troubleshooting/handling-node-failures>` for details. 
+.. caution::
+
+    The procedure described below applies to clusters where Raft-based topology 
+    is enabled. Raft-based topology is automatically enabled in new clusters.
+
+    If you've upgraded an existing cluster from version 5.4, ensure that you 
+    :doc:`manually enabled Raft-based topology </upgrade/upgrade-opensource/upgrade-guide-from-5.4-to-6.0/enable-raft-topology>`.
+    Without Raft-based topology enabled, you must consider the following limitations 
+    while following the procedure:
+
+    * You cannot add new nodes to the cluster if any nodes are down.
+    * You can only bootstrap one node at a time. You need to wait until the status 
+      of one new node becomes UN (Up Normal) before adding another new node.
+    * If the node starts bootstrapping but fails in the middle, for example, due to 
+      a power loss, you can retry bootstrap by restarting the node. If you don't want to
+      retry, or the node refuses to boot on subsequent attempts, consult the 
+      :doc:`Handling Membership Change Failures </operating-scylla/procedures/cluster-management/handling-membership-change-failures>`
+      document. 
 
 Prerequisites
 -------------
 
-#. Before adding the new node, check the node's status in the cluster using :doc:`nodetool status </operating-scylla/nodetool-commands/status>` command. You cannot add new nodes to the cluster if any nodes are down.
+Quorum of Nodes
+=================
 
-    For example:
+Adding a new node requires at least a quorum of nodes in a cluster to be available. 
```

Comment:
This entire section (Quorum of Nodes) applies to all operations like adding and removing nodes/DC.
We can extract it to include and slightly update it to be generic (see below)

```suggestion
Updating the cluster topology requires at least a quorum of nodes in a cluster to be available. 
```